BHoM_Engine/Query/Hash.cs

```diff
@@ -37,14 +37,24 @@
 
 namespace BH.Engine.Base
 {
-    public static partial class Compute
+    public static partial class Query
     {
         /***************************************************/
         /**** Public Methods                            ****/
-        /***************************************************/ 
+        /***************************************************/
+
+        [Description("Computes a Hash code for the iObject. The hash uniquely identifies the iObject based on its properties and their values.")]
```

Comment:
```suggestion
        [Description("Computes a Hash code for the iObject. The hash uniquely represents an object's state based on its combined properties and their values.")]
```

Reply:
@alelom - also suggested change to description. Wonder if this use of the work "state" here will help going forward in explaining use vs guid?
Happy to get your alternate thoughts